main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "Planet"
+require_relative "Solar_System"
+
+def main
+  solar_system = SolarSystem.new("Solus")
+  neptune = Planet.new("Neptune", "blue", 1.02E26, 4.495E9, "greek god of the sea Poseidons planet")
+  mars = Planet.new("Mars", "red", 6.3E23, 227.9E9, "greek god of war Aries planet")
+  venus = Planet.new("Venus", "pink", 4.86E24, 108E9, "greek goddess of love Aphrodites planet")
+  solar_system.add_planet(neptune)
+  solar_system.add_planet(mars)
+  solar_system.add_planet(venus)
+  list = solar_system.list_planets
+
+  found_planet = solar_system.find_planet_by_name("Venus")
+
+  input = ""
+
+  until input == "exit"
+    puts "what would you like to do? list planets, planet details, add planet, exit"
+    input = gets.chomp
+    if input == "list planets"
+      list = solar_system.list_planets
+      puts list
+    elsif input == "planet details"
+      puts "what planet do you want to learn about?"
+      user_add_planet = gets.chomp
+      chosen_planet = solar_system.find_planet_by_name(user_add_planet)
+      puts chosen_planet.summary
+    elsif input == "add planet"
+      planet_new = add_new_planet
+      solar_system.add_planet(add_new_planet)
+    else
```

Comment:
On line 30 you call `add_new_planet` and assign it to the `planet_new` variable. Then on line 31, instead of using `planet_new`, you call `add_new_planet` again! That means the user has to enter all this info twice.

Reply:
THANK YOU for pointing that out. that was driving me bananas i couldnt figure out why it asked twice.